src/pks/surface_balance/constitutive_relations/land_cover/pet_priestley_taylor_evaluator.cc

```diff
@@ -44,7 +44,7 @@ psychrometricConstant(double lh_vap, double elev)
 double
 vaporPressureSlope(double temp_air)
 {
-  return 4098 * Relations::SaturatedVaporPressure(temp_air) / std::pow(temp_air - 35.85, 2);
+  return 4.098 * Relations::SaturatedVaporPressure(temp_air) / std::pow(temp_air - 35.85, 2); //kPa
```

Comment:
rather than making this kPa, can you make the psychrometric constant in Pa?  I would prefer to keep all ATS internals in SI units.